src/components/common/NetworkSelector/index.tsx

```diff
@@ -76,7 +76,9 @@ const NetworkSelector = (): ReactElement => {
       })}
     </Select>
   ) : (
-    <Skeleton width={94} height={31} />
+    <Box px={2}>
+      <Skeleton width={94} height={31} />
+    </Box>
```

Comment:
A Box just to add a padding? Maybe just use sx on the Skeleton?

Reply:
Nice! Worked without the Box and instead margin on the Skeleton.